Dockerfile

```diff
@@ -1,12 +1,13 @@
 ### Stage 0 - Base image
-FROM node:10-alpine as base
+FROM node:12-alpine as base
 ARG npm_token
 ENV NEXUS_TOKEN=$npm_token
 WORKDIR /app
 RUN apk --no-cache update && \
     apk --no-cache upgrade && \
     apk add --no-cache --virtual .build-dependencies python make g++ && \
     npm install -g npm@latest && \
+    rm -rf /usr/local/lib/node_modules/npm/node_modules/node-gyp/test && \
```

Comment:
Better remove the entire npm folder as it is not needed to run on production
You can do this in the `production` target (just after the removal of @godaddy)